bam_tview_curses.c

```diff
@@ -110,6 +110,33 @@ static void curses_clear(struct AbstractTview* tv)
     clear();
     }
 
+static int curses_init_colors(int inverse)
+{
```

Comment:
Yes yes I know... the indentation is sane.  Sorry for not following the local code conventions. :-)